docs/dev/building-on-zksync/useful-address.md

```diff
@@ -18,9 +18,9 @@ Here are some contract addresses that may be useful and/or of interest to those
 | l1_allow_list_addr            | [0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA](https://etherscan.io/address/0x185e050EC1c22c45445a9AdEA99B12e1A1372eBA#code)|
 
 
-## Root hashes
```

Comment:
Why we have this? It is not related to the user and hard to maintain, probably just remove the section?

Reply:
thought that this was something that remains "forever"... agree, lets remove then